apps/jobs/views.py

```diff
@@ -51,22 +56,25 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
     if request.method == 'GET':
-        # getting participant team object for the user for a particular challenge.
+        # getting participant team object for the user for a particular
```

Comment:
I think these changes are because of running autopep8. Can you make sure that these are removed from the PR ?